openpype/plugins/publish/extract_review.py

```diff
@@ -1044,6 +1040,11 @@ def audio_args(self, instance, temp_data, duration_seconds):
                 path_to_subprocess_arg(audio["filename"])
             ))
 
+        # Add exclusive mapping for audio channels
+        audio_in_args.append("-map 0:v")
+        for i in range(len(audio_inputs)):
+            audio_in_args.append(f"-map {i+1}:a")
```

Comment:
This code is OP core and might be executed by python2

Reply:
ok replaced by a classic "{}".format()